### ISSUE TITLE
window notifiaction

### ISSUE THREAD

**Issue by mwoz123** [hovancik/later-on#2]
First of all, that's for the new project.

Can it have the option to show window popup (not only system notifications), and postpone it?

or do you plan to leave it only in sys notifications?

**Comment by hovancik** [comment]
Hi, right now my plan is to "get out of alpha", meaning to make this user friendly...  So I am running it now myself and thinking where to move this. 

I noticed times get out of sync when I lock screen, for example.

So i will make some refactoring of how it works now, try to at least show created reminders at some screen/tray with times of next occurrence and validate for errors. Probably make release at that point. Then add edit of reminders via UI. Features like start at boot, following of DND and such. 

Regarding postponing, that would not be possible with native notifications, as from what I understand only macOS allows buttons in Notification, so that would need custom windows. But first it makes sense for me to allow users create reminders via UI and so on.

**Comment by hovancik** [comment]
I've added some UI :)

**Comment by mwoz123** [comment]
Yeah thanks I know it will be a cron UI but I was rather thinking at poping out window at notification time... 

Maybe with option to delay as it's currently done in Stretchtly (then it not anymore simple cron UI;)
